desci-server/src/controllers/nodes/documents.ts

```diff
@@ -1,26 +1,27 @@
+import { DocumentId } from '@automerge/automerge-repo';
 import { Response } from 'express';
 
 import { prisma } from '../../client.js';
 import { logger } from '../../logger.js';
 import { RequestWithNode } from '../../middleware/authorisation.js';
-import { NodeUuid } from '../../services/manifestRepo.js';
+import { ManifestActions, NodeUuid } from '../../services/manifestRepo.js';
 import repoService from '../../services/repoService.js';
 import { getLatestManifest } from '../data/utils.js';
-// import { ResearchObjectDocument } from '../../types/documents.js';
-// import { backendRepo } from '../../repo.js';
 
 export const getNodeDocument = async function (req: RequestWithNode, response: Response) {
   try {
     logger.info({ userId: req.user.id, uuid: req.node.uuid }, '[START] GetNodeDocument');
-    // response.status(403).send({ ok: false, message: 'This api is deprecated' });
-    // const repo = backendRepo;
 
     const node = req.node;
+    const manifest = await getLatestManifest(node.uuid!, req.query?.g as string, node);
 
-    const manifest = await getLatestManifest(node.uuid, req.query?.g as string, node);
+    if (!manifest) {
+      response.status(400).send({ ok: false, message: 'Node could not be migrated to draft, No manifest found' });
```

Comment:
I feel this phrase is slightly confusing, as `getNodeDocument` is a request for a document, but the error talks about a migration from Node to draft.

Perhaps `Could not get manifest for node` would work? Or is the function name a bit too specific?